Makefile

```diff
@@ -119,7 +123,10 @@ OCI_REGISTRY := projects.registry.vmware.com/tce
 
 ##### LINTING TARGETS
 .PHONY: lint mdlint shellcheck check yamllint misspell actionlint urllint imagelint
-check: ensure-deps lint mdlint shellcheck yamllint misspell actionlint urllint imagelint
+check: check-parallel
+
+check-parallel:
+	$(MAKE) --jobs=$(NUMBER_OF_CORES) --output-sync=target lint mdlint shellcheck yamllint misspell actionlint urllint imagelint
```

Comment:
Running
```
❯ make --jobs=12 --output-sync=target urllint
```
locally _doesn't_ give any of the output until I hit `<ctrl-c>` or it finishes. I noticed similar behavior with `yamllint` and `imagelint`.

Reply:
I totally forgot to mention this in the PR.

In order not to go crazy, I added `--output-sync=target` which basically holds all the output for a given sub-target until the entire target is complete. This is needed because without this option all the output for all the concurrent jobs would be interleaved and show up all at once as lines for each job was emitted. That would make it appear like it's in a hung state.

Reply:
Looks like this is an issue for macOS users. The BSD make on Mac gives an error with this:

```
/Library/Developer/CommandLineTools/usr/bin/make: unrecognized option `--output-sync=target'
```

Reply:
Yea, we can see if brew has a newer version. If they do, we can detect the version and make decisions based on that.

Reply:
Looks like the newer version is available, it's just not the default: https://stackoverflow.com/questions/38901894/how-can-i-install-a-newer-version-of-make-on-mac-os

Reply:
Added a check for this

Reply:
Also dropped a line about the suppressed output